EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc

```diff
@@ -160,6 +163,9 @@ void GEMRawToDigiModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
     for (auto amcData : *(amc13Event->getAMCpayloads())) {
       uint16_t amcBx = amcData.bx();
       uint8_t amcNum = amcData.amcNum();
+      // for the GE1/1-ME1/1 trigger test stand, the AMC13 number is fixed to 3!
+      if (useB904Data_)
+        amcNum = 3;
```

Comment:
@dildick This is not a nice way to do this. As mentioned in the email, it is better to make a new EMAP.

Reply:
Alright. I can leave this out.